components/renku_data_services/data_connectors/db.py

```diff
@@ -215,7 +215,10 @@ async def update_data_connector(
 
         current_etag = data_connector.dump().etag
         if current_etag != etag:
-            raise errors.ConflictError(message=f"Current ETag is {current_etag}, not {etag}.")
+            raise errors.ConflictError(
+                message=f"Current ETag is {
+                                       current_etag}, not {etag}."
+            )
```

Comment:
The format doesn't look correct.

Reply:
Done in 968df0f89befff1c2e5564770fdccd0638479432.